src/styles/utils/cycles.scss

```diff
@@ -0,0 +1,28 @@
+@each $day, $index in $days-of-week {
+  .calendar--start-day-#{$day} {
+    .calendar__day:first-child {
+      margin-left: calc(#{$cell-size} * #{$index} + $gap * #{$index});
```

Comment:
The use of multiple operations inside the `calc` function can be simplified. Consider storing the result of `#{$cell-size} * #{$index} + $gap * #{$index}` in a variable for better readability and maintainability.